code/modules/mob/living/carbon/human/human.dm

```diff
@@ -201,38 +201,46 @@
 
 	var/b_loss = null
 	var/f_loss = null
+	var/armor = max(0, 1 - getarmor(null, "bomb"))
 	switch(severity)
 		if(1)
-			b_loss += rand(125, 175)
-			if(!prob(getarmor(null, "bomb") + 75)) //Much less likely to gib than before
-				gib()
-				return
-			else
-				var/atom/target = get_edge_target_turf(src, get_dir(src, get_step_away(src, src)))
-				throw_at(target, 200, 4)
-		if(2)
-			b_loss += rand(50, 70)
-			f_loss += rand(50, 70)
+			b_loss += rand(120, 160) * armor	//Probably instant death
+			f_loss += rand(120, 160) * armor	//Probably instant death
 
-			if(prob(getarmor(null, "bomb")))
-				b_loss = b_loss/1.5
-				f_loss = f_loss/1.5
+			var/atom/target = get_edge_target_turf(src, get_dir(src, get_step_away(src, src)))
+			throw_at(target, 200, 4)
 
 			if(!istype(wear_ear, /obj/item/clothing/ears/earmuffs))
-				ear_damage += 30
-				ear_deaf += 120
-			if(prob(70))
-				KnockOut(10)
+				ear_damage += 60 * armor
+				ear_deaf += 240 * armor
+
+			adjust_stagger(round(4 * armor,0.1))
+			add_slowdown(round(4 * armor,0.1))
+			KnockOut(10 * armor)
```

Comment:
While `KnockOut` is realistic, I'd personally prefer we moved away from it in favor of `KnockDown` here. Let pain or crit handle losing consciousness instead.
```suggestion
			KnockDown(10 * armor)
```
Maybe a little feedback if they are still `CONSCIOUS` to inform them they'll be out of commission for a bit, as well, would be good.
Something like
```
if(stat)
	tochat(... "You fail to see the plate number of the truck that hit you.")
```